### PR TITLE
[#38941] hotel pages

### PR DESCRIPTION
**Pull request by trunghq-2593**
## Related Tickets
- [#38941](https://edu-redmine.sun-asterisk.vn/issues/38941)

## HOW
- cut layouts for page hotel
- show data frontend
Screenshot from 2021-08-05 08-54-33

### CODE REVIEW

**Comment by trunghq-2593** [comment]
ready

**Comment by trunghq-2593** [comment]
ready

File: resources/lang/vn/customer.php [hunk 1]
```diff
@@ -16,4 +16,13 @@
...
     'travel_agency' => 'Quản lý khách sạn',
+    'star_rating' => 'Đánh giá',
+    'name' => 'Tên',
+    'price' => 'Giá',
+    'ascending' => 'Tăng dần',
+    'descending' => 'Giảm dần',
+    'sort_by' => 'Sắp xếp',
+    'to' => 'Đến',
+    'book' => 'Đặt',
+    'night' => 'night',
```
Comment (ghost) [inline resources/lang/vn/customer.php]: sao night ko dịch à?

File: resources/views/customer/layouts/index/content.blade.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+            <a class="book-icon" href="#"></a>
+        </div>
+        <div class="itemlabel">
+            <button class="bookbtn right mt1">@lang('customer.book')</button>
+            <b>{{ $hotel->name }}</b><br/>
+            <p class="lightgrey"><span class="green size14"><b>${{ $hotel->avg_price }}</b></span> /@lang('customer.night')</p>
+        </div>
+    </div>
+    @endforeach
+    @endif
```
Comment (ghost) [inline resources/views/customer/layouts/index/content.blade.php]: indent 4 spaces

**Comment by trunghq-2593** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**